examples/test-app/steps/Country/index.js

```diff
@@ -1,14 +1,27 @@
 const { Question, form, field, goTo, branch } = require('@hmcts/one-per-page');
```

Comment:
Why have we reverted back to index.js, surely this should be called Country.js? Michael was keen to move away from index.js and the the old Divorce framework naming convention. Since we are dealing with files that contain classes the name of the file should match the name of the class.

Reply:
it made sense when all js files were in same folder, however now i have each class is contained within its own folder with content and view it makes less sense. The main reason being that when you require a class currently you can do `require('steps/country')`, but if we rename the js files to classname we would have to do `require('steps/country/Country)`

Reply:
I see your point. There is another issue that we'll encounter and that's searching for files (e.g. Steps) - they are called the same name index.js :-(

Reply:
sure but you have the reference in the file path in error stacks etc if thats you point

Reply:
I wasn't referring to stack traces but generally searching for a particular file by it's name.

Reply:
yea thats a good argument, not sue the best way to approach this one, as i dont like idea of having to `require('steps/country/Country)`

Reply:
Personally I don't see that as an issue, it's no different (coming from Java) in importing a Java class/interface `import java.lang.annotation.Annotation` everyone knows what the path is and what the class is. 
https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/Annotation.html

Reply:
I don't want to hold this up, I'll leave it up to you. I've had my say :-)